modin/engines/ray/utils.py

```diff
@@ -25,3 +29,105 @@ def handle_ray_task_error(e):
                 else:
                     raise att_err
     raise e
+
+
+# Register a fix import function to run on all_workers including the driver.
+# This is a hack solution to fix #647, #746
+def _move_stdlib_ahead_of_site_packages(*args):
+    site_packages_path = None
+    site_packages_path_index = -1
+    for i, path in enumerate(sys.path):
+        if sys.exec_prefix in path and path.endswith("site-packages"):
+            site_packages_path = path
+            site_packages_path_index = i
+            # break on first found
+            break
+
+    if site_packages_path is not None:
+        # stdlib packages layout as follows:
+        # - python3.x
+        #   - typing.py
+        #   - site-packages/
+        #     - pandas
+        # So extracting the dirname of the site_packages can point us
+        # to the directory containing standard libraries.
+        sys.path.insert(site_packages_path_index, os.path.dirname(site_packages_path))
+
+
+# Register a fix to import pandas on all workers before running tasks.
+# This prevents a race condition between two threads deserializing functions
+# and trying to import pandas at the same time.
+def _import_pandas(*args):
+    import pandas  # noqa F401
+
+
+def initialize_ray(cluster=None, redis_address=None, redis_password=None):
```

Comment:
I don't see the `cluster` or `redis_address` being passed in. Why do we need these as parameters?

Reply:
Please also add docstring.